lib/gulp-image-converter.js

```diff
@@ -0,0 +1,65 @@
+var path = require('path');
+var _ = require('underscore');
+var through = require('through2');
+var lwip = require('lwip');
+var File = require('vinyl');
```

Comment:
Ah good catch, it was being used but I stopped using it in https://github.com/toolness/teach.webmaker.org/commit/2fa210f2cd94614eac0a03be242cb87784eae427 and forgot to remove the `require`.